app/models/creatorship.rb

```diff
@@ -1,6 +1,6 @@
 class Creatorship < ApplicationRecord
-  belongs_to :pseud, inverse_of: :creatorships
-  belongs_to :creation, inverse_of: :creatorships, polymorphic: true, touch: true
+  belongs_to :pseud
```

Comment:
I'm not having a lot of luck googling here -- do you have info about the change that allows us to remove `inverse_of`?

Reply:
It's a couple different things, so not the easiest to Google unfortunately :disappointed: The first bit is:
> Starting with Rails 4, the inverse_of setting is implicit between these [`has_many` and `belongs_to`] models, allowing you to traverse them bidirectionally.

However, that only worked in 1 direction prior to Rails 6.1 (I'd recommend the example in https://lilyreile.medium.com/rails-6-1-new-framework-defaults-what-they-do-and-how-to-safely-uncomment-them-c546b70f0c5e#4e5e or https://www.bigbinary.com/blog/rails-6-1-adds-support-for-belongs_to-to-has_many-inversing).

The next bit is that Rails 6.1 fixed the behavior to be traverse both directions.

The secret third thing is that we seem to run into https://github.com/rails/rails/issues/40867 with these (or something similar at least), so keeping the `inverse_of` will actually break things...

Reply:
Ah, okay, thanks! I did read about `has_many_inversing` while I was looking for an explanation, but I dismissed that as not applicable based on a number of silly assumptions.